src/material/input/input.ts

```diff
@@ -197,6 +198,25 @@ export class MatInput
     if (!this._isTextarea && getSupportedInputTypes().has(this._type)) {
       (this._elementRef.nativeElement as HTMLInputElement).type = this._type;
     }
+
+    // In blink and webkit browsers a focused number input does not increment or decrement the value
+    // on mouse wheel interaction unless a wheel event listener is attached to it or one of its ancestors or a passive wheel listener is attached somewhere in the DOM.
+    // For example: Hitting a tooltip once enables the mouse wheel input for all number inputs as long as it exists.
+    // In order to get reliable and intuitive behavior we apply a wheel event on our own
+    // thus making sure increment and decrement by mouse wheel works every time.
+    if (
```

Comment:
Consider moving this out into a private method since the setter is getting pretty big.